support/openharmony/entry/src/main/ets/pages/Index.ets

```diff
@@ -48,8 +52,11 @@ struct Index {
     type: XComponentType.SURFACE,
     libraryname: 'servoshell',
   }
-  @State urlToLoad: string =  'https://servo.org'
+
   private context = getContext(this) as common.UIAbilityContext;
+  @LocalStorageProp('InitialURI') InitialURI: string = "unused"
+  @LocalStorageProp('CommandlineArgs') CommandlineArgs: string = ""
```

Comment:
Is there documentation on the `@LocalStorageProp` decorator? Why do we need this declaration and how does it relate to the code in EntryAbility's `onWindowStageCreate` method where we pass `this.init_params` 

```
windowStage.loadContent('pages/Index',this.init_params,
```

Reply:
We need to use one of the statemanagement functions to share data between the ability and the page.
Localstorage seems like the appropriate type, since we only expect to have on Ability, and don't need any cross-ability state sharing. 
We then also have the choice between `@LocalStorageProp` and  `@LocalStorageLink`, where the former is one-way synced from ability to page, and the latter provides 2-way synchronization.
We currently only need the localstorage to provide initial values to servo, so `@LocalStorageProp` is sufficient.

If we ever need / want to save state or synchronize settings across devices, then the whole settings approach would need to be revisited.

Documentation links: 
[Application state management overview](https://docs.openharmony.cn/pages/v5.0/en/application-dev/quick-start/arkts-application-state-management-overview.md), 
[Localstorage documentation](https://docs.openharmony.cn/pages/v5.0/en/application-dev/quick-start/arkts-localstorage.md), 
[App storage documentation](https://docs.openharmony.cn/pages/v5.0/en/application-dev/quick-start/arkts-appstorage.md)